.github/workflows/deploy_website.yml

```diff
@@ -0,0 +1,38 @@
+name: Deploy website
+
+on:
+   push:
+    branches:
+      - master
+
+jobs:
+  deploy:
+
+    runs-on: ubuntu-latest
+
+    steps:
+    - uses: actions/checkout@v2
+    - name: Deploy website if diff exists
+      env:
+        GITHUB_TOKEN: ${{ secrets.WEBSITE_GITHUB_TOKEN }}
+      run: |
+        SUBDIR=website
+        git fetch
+        REV=$(git log -5 --pretty=oneline origin/gh-pages | grep "Deploy website" |  awk 'NF>1{print $NF}'  | head -1)
+        # This condition passes in 2 conditions:
+        # 1. The revision does not exist (squash/force push happened)
+        # 2. There are changes between last deployed revision and HEAD
+        if [[ ! $(git rev-parse --verify -q "$REV^{commit}") ||  $(git diff-index $REV -- $SUBDIR) ]]; then
+          echo "Changes detected in directory $SUBDIR between origin/master and this diff"
+
+          cd $SUBDIR
+          yarn --no-progress
+
+          git config --global user.email omry@users.noreply.github.com
+          git config --global user.name omry
+          echo "machine github.com login docusaurus-bot password $WEBSITE_GITHUB_TOKEN" > ~/.netrc
```

Comment:
I had to rename $GITHUB_TOKEN to $WEBSITE_GITHUB_TOKEN because github secrets cannot start with "GITHUB" (reserved prefix.)